File: Person.py
class Person:

# __init__(newName="none", address="none", phone="999-999-9999") â—¦ initializes a Person object ------- 5 points
    def __init__(self, newName="none", address="none", phone="999-999-9999"):
        self.name = newName
        self.address = address
        self.phone = phone

# setName(string) -------- sets the name of the object to be equal to the string parameter provided
    def setName(self, name):
        self.name = name

# getName()â†’ string 1 point â—¦ returns the name of the Person as a string
    def getName(self):
        return(self.name)

# getPhone()â†’ string 1 point â—¦ returns the phone number of the Person as a string
    def getPhone(self):
        return(self.phone)
    
#str method needed according to gradescope
    def __str__(self):
        stringToReturn = ("Name: " + self.name + "\nAddress: " + self.address + "\nPhone Number: " + self.phone)
        return stringToReturn

File: Person.java
public class Person {
    public String name;
    public String address;
    public String phone;
    Person(String newName, String newAddress, String newPhone){
        this.name = newName;
        this.address = newAddress;
        this.phone = newPhone;
    }
    //setname
    void setName(String newName){
        this.name = newName;

    }
    //getname
    String getName(){
        //CHANGED FROM OG, USE PARENTHESES WITH RETURN
        return(this.name);
    }
    //setaddress
    void setAddress(String newAddress){
        this.address = newAddress;
    }
    //getaddress
    String getAddress(){
        //CHANGED FROM OG, USE PARENTHESES WITH RETURN
        return(this.address);
    }
    //setphone
    void setphone(String newPhone){
        this.phone = newPhone;
    }
    //getphone
    String getphone(){
        //CHANGED FROM OG, USE PARENTHESES WITH RETURN
        return(this.phone);
    }
    //tostring --- like str in python
    public String toString(){
        return("The name of the person is " + this.name + ". Their address is " + this.address + "and their phone is " + this.phone + ".");
    }
}
